app/src/main/java/org/fossasia/openevent/general/utils/Utils.kt

```diff
@@ -0,0 +1,33 @@
+package org.fossasia.openevent.general.utils
+
+import android.content.Context
+import android.graphics.BitmapFactory
+import android.net.Uri
+import android.support.customtabs.CustomTabsIntent
+import android.support.v4.content.ContextCompat
+import android.view.View
+import kotlinx.android.synthetic.main.fragment_profile.view.*
+import org.fossasia.openevent.general.R
+
+object Utils {
+
+    fun setUpCustomTab(context: Context, url: String) {
+        var URL = url
+        if (!url.startsWith("http://") && !url.startsWith("https://")) {
+            URL = "http://$url"
+        }
+
+        CustomTabsIntent.Builder()
+                .setToolbarColor(ContextCompat.getColor(context, R.color.colorPrimaryDark))
+                .setCloseButtonIcon(BitmapFactory.decodeResource(context.resources, R.drawable.ic_arrow_back_white_cct_24dp))
+                .setStartAnimations(context, R.anim.slide_in_right, R.anim.slide_out_left)
+                .setExitAnimations(context, R.anim.slide_in_left, R.anim.slide_out_right)
+                .build()
+                .launchUrl(context, Uri.parse(URL))
+    }
+
+    fun showProgressBar(rootView: View, show: Boolean) {
```

Comment:
Did you run the app locally ? This function is not same everywhere

Reply:
It won't work in an activity. I didn't include this in any activity files

Reply:
Not good, pass progressBar, not view